packages/docusaurus-plugin-openapi-docs/src/openapi/types.ts

```diff
@@ -197,6 +197,7 @@ export interface ParameterObject {
   param?: Object;
   // ignoring stylings: matrix, label, form, simple, spaceDelimited,
```

Comment:
Yeah, it's on the roadmap to tackle param serialization styles, but this should be ok for now.